sycl/test/basic_tests/esimd/check_compile.cpp

```diff
@@ -0,0 +1,10 @@
+// RUN: %clangxx -fsycl -fsycl-explicit-simd -fsycl-device-only -fsyntax-only -Xclang -verify %s
```

Comment:
We don't usually use clang driver in FE tests.
```suggestion
// RUN: %clangxx -cc1 -fsycl -fsycl-is-device -fsycl-explicit-simd -fsyntax-only -verify %s
```